packages/cli/src/commands/graph/tasks/graphOrderTask.ts

```diff
@@ -23,22 +23,35 @@ export function graphOrderTask(
     options: { persistentOutput: true },
     async task(ctx: GraphCommandContext, task) {
       let order: { packages: PackageEntry[] };
-      const { basePath, output } = options;
+      const { srcDir, output, basePath } = options;
       let selectedPackageName: string;
 
       if (process.env['TEST'] === 'true') {
         // Do this on the main thread because there are issues with resolving worker scripts for worker_threads in vitest
         const { intoGraphOutput } = await import('./graphWorker.js').then((m) => m);
         const { getMigrationOrder } = await import('@rehearsal/migration-graph').then((m) => m);
         // eslint-disable-next-line @typescript-eslint/no-unsafe-argument, @typescript-eslint/no-unsafe-call
-        order = intoGraphOutput(getMigrationOrder(basePath), basePath);
+        order = intoGraphOutput(
+          getMigrationOrder(srcDir, {
+            basePath,
```

Comment:
How about, instead of `basePath` (root package for monorepos) use `rootPath` (always root or repo) and `packagePath` (or `[workspacePath](https://docs.npmjs.com/cli/v7/using-npm/workspaces)`) for the current target package?

For single-repo `packagePath` will be equals to `rootPath`... but it will be clear where do we expect to use root (report, configs) and where current package dir.

@chadhietala @lynchbomb ?

Reply:
Yea I'm not happy with the naming of this stuff. I'm going to come back around and cleanup this stuff once the tests are green.